build.gradle.kts

```diff
@@ -61,7 +69,42 @@ kotlin {
     }
 }
 
+mavenPublishing {
+    publishToMavenCentral(SonatypeHost.S01)
```

Comment:
@rubensf - This is what we would change if we wanted to publish to central instead of s01.